### DISCUSSION TITLE
Acknowledgement list

### DISCUSSION THREAD

**Discussion by iorii9** [tonmetaspace/.github#35] [Proposals]
_**Organizations:**_
Mozilla hubs
Webspace
Webaverse
Telegram
The Open Network
Github
Open AI (Chat GPT)
TON CON 2022 team
MIR.one
TON Fingerprints
TON Svet

**_Personalities:_**
Gragory Slynko
Gleb Bakharwv
Yaroslav 
Oxana Zlobina
Roman Inozemtsev
Iurii Iakovlev
Marat Minkin
Sergey Suvorov
Mr. Proton
Robin
Justin Hyun
Steve Yun
Sarah
Vladimir @ProstoDirektor
Evgenii Gudylev

**Comment by kerrymatten** [comment]
where?
